deploy/examples/cluster.yaml

```
@@ -281,6 +281,21 @@ spec:
     # No values or 0 means that the operator will wait until the placement groups are healthy before unblocking the next drain.
     pgHealthCheckTimeout: 0
 
+  # csi defines CSI Driver settings applied per cluster.
+  csi:
+    readAffinity:
+      # Set enabled to true to enable read affinity.
```

Let's tell them basically about the feature so they don't always have to go to the doc.
```suggestion
      # Enable read affinity to enable clients to optimize reads from an OSD in the same topology.
```